src/Views/Wallpaper.vala

```diff
@@ -122,10 +124,12 @@ public class PantheonShell.Wallpaper : Gtk.Box {
         actionbar.pack_end (color_button);
         actionbar.pack_end (combo);
 
-        orientation = VERTICAL;
-        append (separator);
```

Comment:
It seems like the separator isn't used at all anymore so you can clean up that object.

Reply:
Oops thank you!